lib/talkable/railtie.rb

```diff
@@ -5,7 +5,7 @@ class Railtie < Rails::Railtie
     end
 
     initializer "talkable.add_middleware" do |app|
-      app.middleware.use Talkable::Middleware
+      app.middleware.insert_after Rack::TempfileReaper, Talkable::Middleware
```

Comment:
You sure this middleware is always present? I don’t see it in our list.
```sh
$ rails middleware
use Webpacker::DevServerProxy
use UTF8Cleaner::Middleware
use Rack::Sendfile
use ActionDispatch::Static
use ActionDispatch::Executor
use Rack::Rewrite
use Rack::Runtime
use Rack::MethodOverride
use ActionDispatch::RequestId
use RequestStore::Middleware
use ActionDispatch::RemoteIp
use HandleUnknownHttpMethod
use Sprockets::Rails::QuietAssets
use Rails::Rack::Logger
use ActionDispatch::ShowExceptions
use ActionDispatch::DebugExceptions
use Bugsnag::Rack
use ActionDispatch::Reloader
use ActionDispatch::Callbacks
use ActionDispatch::Cookies
use ActionDispatch::Session::CookieStore
use ActionDispatch::Flash
use Rack::Head
use Rack::ConditionalGet
use Rack::ETag
use RequestFilter
use EtagCaching
use ThrottlingMiddleware
run Talkable::Application.routes

Reply:
so far `Rack::ETag` seems to be a good candidate to hook to.

Reply:
lets use `defined?(::Warden::Manager)` in order to explicitly specify the order of middlewares